src/components/styled/StyledSelect.js

```diff
@@ -0,0 +1,3 @@
+import styled from 'styled-components';
+
+export const StlyedSelect = styled.select``;
```

Comment:
no one needs empty styled components ;) 
If you dont need it, dont code it :)

Reply:
Thanks for the advice.